bhmdoc/CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ Versioning](https://doc.rust-lang.org/cargo/reference/semver.html).
 
 ## [Unreleased]
 
+### Changed
+
+- `bh-jws-utils` dependency updated from 0.5 to 0.6
+
+### Added
+
+- added `From<DateTime> for chrono::DateTime<Utc>`
+
```

Comment:
There already is the `Added` section. Put your bullet there. Also, make it a full sentence, i.e., capitalized first word and a dot at the end.